docs/user/README.md

```diff
@@ -29,14 +29,21 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
    - Run Tests: `cargo test`
    - Run DBMS Server: `cargo run --bin modelardbd path_to_local_data_folder`
    - Run Client: `cargo run --bin modelardb [server_address] [query_file]`
-5. Move `modelardbd`, `modelardbm`, and `modelardb` from the `target` directory to any directory.
+5. Move `modelardbd`, `modelardbm`,`modelardb`, and `modelardbb` from the `target` directory to any
+   directory.
+6. Install and test the Python bindings for `modelardb_embedded` using Python:
+   * Install: `python3 -m pip install .`
+   * Run Tests: `python3 -m unittest`
 
 ## Usage
-ModelarDB consists of three binaries: `modelardbd` is a DBMS server that manages data and executes SQL queries,
-`modelardbm` is a manager for one or more DBMS servers deployed on the *edge* or in the *cloud*, and `modelardb` is a
-command-line client for connecting to a DBMS server and executing commands and SQL queries. `modelardbd` uses local
-storage on the edge and an Amazon S3-compatible or Azure Blob Storage object store in the cloud. `modelardbd` can be
-deployed alone or together with `modelardbm` depending on the use cases.
+ModelarDB consists of four binaries and a library with bindings: `modelardbd` is a DBMS server that manages data and
+executes SQL queries, `modelardbm` is a manager for one or more DBMS servers deployed on the *edge* or in the *cloud*,
```

Comment:
would it be better to say "`modelardbd` instances" instead of "DBMS servers" here?

Reply:
Fixed.